src/utils/ipfs.ts

```diff
@@ -0,0 +1,44 @@
+import { CowError } from './common'
+import { Ipfs } from './context'
+
+type PinataPinResponse = {
+  IpfsHash: string
+  PinSize: number
+  Timestamp: string
+}
+
+export async function pinJSONToIPFS(
+  file: any,
```

Comment:
we may follow the ESLint suggestion here and replace the `file: any` to `file: unknown`

> Use `unknown` instead, this will force you to explicitly, and safely assert the type is correct.

Reply:
Didn't work with `unknown`?